src/cdk/commands/aslVisualizationCDKManager.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { getLogger, Logger } from '../../shared/logger'
+import { AslVisualizationCDK } from './aslVisualizationCDK'
+import { ConstructNode } from '../explorer/nodes/constructNode'
+import { renderGraphCommand } from './renderGraph'
+import { StateMachineGraphCache } from '../../../src/stepFunctions/utils'
+import { AbstractAslVisualizationManager } from '../../../src/stepFunctions/commands/visualizeStateMachine/abstractAslVisualizationManager'
+
+export class AslVisualizationCDKManager extends AbstractAslVisualizationManager {
+
+    public constructor(extensionContext: vscode.ExtensionContext) {
+        super(extensionContext)
+    }
+
+    public async visualizeStateMachine(
+        globalStorage: vscode.Memento,
+        node: ConstructNode
+    ): Promise<vscode.WebviewPanel | undefined> {
+        const logger: Logger = getLogger()
+        const cache = new StateMachineGraphCache()
+        const uniqueIdentifier = node.label
+        const cdkOutPath = node.id?.replace(`/tree.json/${node.tooltip}`, ``)
+        const stackName = node.tooltip?.replace(`/${uniqueIdentifier}`, ``)
+        // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
+        const templatePath = cdkOutPath + `/${stackName}.template.json`
```

Comment:
Can we not just add the cdkOutPath to the template literal and remove the eslint disable comment?